sound/soc/codecs/tfa989x.c

```diff
@@ -76,6 +84,52 @@ static int tfa989x_hw_params(struct snd_pcm_substream *substream,
 					     TFA989X_I2SREG_I2SSR_MASK, sr);
 }
 
+static int tfa989x_get_mode(struct snd_kcontrol *kcontrol, struct snd_ctl_elem_value *ucontrol)
+{
+	struct snd_soc_component *component = snd_soc_kcontrol_component(kcontrol);
+	struct tfa989x *tfa989x = snd_soc_component_get_drvdata(component);
```

Comment:
Just noticed that this variable is kind of annoying to type :D Can we call it `tfa` only maybe?

```suggestion
	struct tfa989x *tfa = snd_soc_component_get_drvdata(component);
```